features/polyExplorer/src/locales/de/infoScreens/dataRegionsDiagramInfoScreen.json

```diff
@@ -1,2 +1,2 @@
 {
-    "p1": "This diagram shows how much data flows from the messengers into different data regions. "}
+    "p1": "Dieses Diagramm zeigt, wieviele Daten von den Messengern in die verschiedenen Datenregionen fließen. "}
```

Comment:
Wäre das nicht eher: "Dieses Diagramm zeigt, mit wie vielen Unternehmen in welchen Ländern die Messenger Daten teilen" - Es betrachtet ja nicht die Menge an Datentypen sondern Unternehmen. Ansonsten muss der Chart bald nochmal angepasst werden